src/handleUrl.js

```diff
@@ -0,0 +1,35 @@
+import globby from 'globby';
+
+export default async function(assets) {
+  const globbyAssets = [];
+  const normalAssets = [];
+  try {
```

Comment:
just let it buble all the way up, and catch it at onvocation time, and push the error into `compilation.errors` array.
https://github.com/SimenB/add-asset-html-webpack-plugin/blob/36cd3046cdd9b46171a9fcebdf4835337f5f33b2/src/addAllAssetsToCompilation.js#L50